index.html

```diff
@@ -267,12 +267,20 @@
   <body>
     <section id='abstract'>
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
-our capability to operate motor vehicles, university degrees assert our level
-of education, and government-issued passports permit travel between countries.
-This specification provides a mechanism to express these sorts of
-[=credentials=] on the Web in cryptographically secure, privacy-respecting,
-and machine-verifiable way.
+This specification provides normative and non-normative guidance on
+implementing and managing [=verifiable credentials=] and associated
+cryptographic practices. It emphasizes the importance of understanding and
```

Comment:
```suggestion
This specification provides a standard and non-normative guidance on
implementing and managing [=verifiable credentials=] and associated
cryptographic practices. It emphasizes the importance of understanding and
```

I am not sure about the "guidance" term here. This is a standard, and the normative items should be emphasized as such.

Reply:
@iherman — Does this cover your concern?
```suggestion
This specification provides normative requirements and non-normative guidance
for implementing and managing [=verifiable credentials=] and associated
cryptographic practices. It emphasizes the importance of understanding and
```

Reply:
Yes, thank you.